alternatingCase/alternatingCase.js

```diff
@@ -0,0 +1,12 @@
+var toUpper = require('../toUpper/toUpper');
+var toLower = require('../toLower/toLower');
+
+String.prototype.alternatingCase = function() {
+  var newString = '';
+  for (var i = 0; i < this.length; i++) {
+    (i % 2 === 0) ? newString += this[i].toLower() : newString += this[i].toUpper();
```

Comment:
I suggest you add a comment on this line to explain what the line of code is achieving